### ISSUE TITLE
Configs

### ISSUE THREAD

**Issue by SilverShadow525** [TheUnderTaker11/GeneticsReborn#49]
When I tried to use the metal Syringe in-game, it said it was disabled in the configs. I look on this site for a solution, and I find out how to change that. However, I have never tried to change any mod configs, so naturally, I dont know how to change the files. Im providing you with two Screenshots that might help you better understand (I cropped out any personal info). The first SS will show you that all my mods are .jar files, and before I knew to get java when downloading mods for the first time, I tried to open my files with Internet Explorer (the suggested program at the time for opening the .jar files). The Second SS will show you what I got by trying to open the file on NotePad. If you can help me find where to change the configs, I know what to do from there. Thank You.

![2017-11-29 2](https://user-images.githubusercontent.com/34088645/33364915-0d4a8d4c-d4ac-11e7-8d1d-1dff60e9167a.png)

![2017-11-29 3](https://user-images.githubusercontent.com/34088645/33364886-ee4da01e-d4ab-11e7-9e40-094142f79ea3.png)

**Comment by alothmanmoe** [comment]
Sorry it took a little while to respond i was a little busy.
As for the config, you do not open the jar file, instead navigate to the config folder(its in the same location the mods folder is in) and you should find the config file in there.

**Comment by SilverShadow525** [comment]
No problem with the wait. Thanks, I'll be sure to try it when I get the chance

**Comment by TheUnderTaker11** [comment]
Not sure why so many people want to do it, but just gonna give you a tip. Never open the .jar unless you already know what you are doing beforehand. If you don't chances are you are doing something wrong. There is sure to be other reasons but only time I've cracked open a .jar is to get out some textures. 
       
Also not to be rude but try googling stuff like this. When troubleshooting it's not a bad idea to think to yourself, "if I had to make a title to post my problem to a forum, what would it be?" Chances are someone before you has made a pretty similar post before somewhere. Just some advise for future problems.
